.env.example

```
@@ -7,5 +7,5 @@ ENV = "development" # Other: production, staging
 QUEUE_NAME = "DISCORD_QUEUE" #Default: "DISCORD_QUEUE"
 RDS_BASE_API_URL = "http://localhost:3000"
 MAIN_SITE_URL = "http://localhost:4200"
-BOT_PRIVATE_KEY = "<PRIVATE_KEY>"
 VERIFICATION_SITE_URL = "http://localhost:3443"
+BOT_PRIVATE_KEY = "<PRIVATE_KEY>"
```

dotenv-linter (3.3.0)

[warning] [SpaceCharacter] The line has spaces around equal sign

(SpaceCharacter)